tests/testthat/test-events.R

```diff
@@ -21,7 +23,7 @@ test_that("create_event_based_processes assigns a listener to each event", {
     states,
     variables,
     events,
-    parameters
+    psq
```

Comment:
this change suggests that this test is incomplete and would have failed if it was used (i.e., the `parameters` promise was never forced). The expectation below seems very minimal; we might add a ticket to expand this later with some indication of what the test here is for